pkg/storage/impl/mysql/mysql_impl.go

```diff
@@ -21,7 +47,7 @@ type MySQLImpl struct {
 
 // implementation of AgentInterface for MySQL
 func NewMySQLImpl() *MySQLImpl {
-	return &MySQLImpl{Name: "mysql", Log: logging.LogInit(logging.CONNECTOR, "StorageManager")}
+	return &MySQLImpl{Name: mysqlAgent, Log: logging.LogInit(logging.CONNECTOR, "StorageManager")}
```

Comment:
I'd suggest to add impl type into the LogInit, so later we will be able easily separate messages from different implementations.

Reply:
ok